index.html

```diff
@@ -79,22 +85,43 @@ <h2 class="backend">Back End Dev</h2>
                         <h2 class="year">2015</h2>
                     </li>
                 </ul>
-            </div>
-            <p class="p-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="framed">
+                    <li>
+                        <h2 class="canopy">CANOPY</h2>
+                    </li>
+                    <li>
+                        <img src="Counter.png" alt="">
+                    </li>
+                    <li>
+                        <h2 class="backend">Back End Dev</h2>
+                    </li>
+                    <li>
+                        <img src="Counter.png" alt="">
+                    </li>
+                    <li>
+                        <h2 class="year">2015</h2>
+                    </li>
+                </ul>
+                <p class="p-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <p class="p-textd">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
I love your design for your dekstop version due to it doesn't matter if you are in a large or medium screen for all desktops your design is responsive, but your mobile design can be better if you add styles to make it more responsive for different mobile screen sizes,

- [ ] [OPTIONAL] Improve the responsive design for your mobile version.

![image](https://user-images.githubusercontent.com/60164239/131208698-79e55033-3711-4030-9425-20437c1624e9.png)